stdlib/src/builtin/string.mojo

```diff
@@ -537,6 +537,22 @@ fn isdigit(c: String) -> Bool:
 # ===----------------------------------------------------------------------=== #
 
 
+@always_inline
+fn isupper(c: String) -> Bool:
+    """Determines whether the given character is an uppercase character.
+
+    This currently only respects the default "C" locale, i.e. returns True iff
```

Comment:
typo, two "f"

Reply:
It's a little esoteric, but "iff" is jargon that means "if and only if": https://www.merriam-webster.com/dictionary/iff

Perhaps we should spell this out fully, since understandably many folks may not know this term and it does look a lot like a typo.

(@arthurevans @scottamain FYI for style guide considerations.)

Reply:
Ok, I see. I left only this comment, to reduce noise